compiler/src/dotty/tools/dotc/reporting/messages.scala

```diff
@@ -2743,8 +2744,15 @@ class MissingImplicitArgument(
         // show all available additional info
         def hiddenImplicitNote(s: SearchSuccess) =
           i"\n\nNote: ${s.ref.symbol.showLocated} was not considered because it was not imported with `import given`."
+        def noChainConversionsNote(s: Iterable[TermRef]): Option[String] =
+          Option.when(s.nonEmpty)(
+              i"\n\nNote: Chaining implicit conversions are not allowed in Scala. " +
+              i"The following conversions are in scope:${s.map(g => s"\n  - ${g.symbol.showDcl}").mkString}" +
```

Comment:
The previous messages stated that these are the conversions that are available but were ignored - not that these conversions are all conversions available in scope. Was there any reason why this got changed?

Reply:
I thought the previous wording might mislead people into thinking that there were implicits found to input into all the listed `Conversion`s, instead of those `Conversion`s simply being all the conversions in scope with the desired result type. Now that you mention it, this new wording does make it seem like it's listing all conversions in scope and not just those with the desired result type. Perhaps `The following conversions resulting in ${pt.show} are in scope` would be better.